ui/app/reducers.js

```diff
@@ -41,12 +41,23 @@ function rootReducer (state, action) {
   return state
 }
 
-window.logState = function () {
+window.logStateString = function (cb) {
   let state = window.METAMASK_CACHED_LOG_STATE
   const version = global.platform.getVersion()
-  state.version = version
-  let stateString = JSON.stringify(state, removeSeedWords, 2)
-  return stateString
+  const browser = window.navigator.userAgent
+  return global.platform.getPlatformInfo((platform) => {
```

Comment:
What if a platform doesn't define `getPlatformInfo`? Can we implement this feature in a backwards-compatible way, that detects the function before calling it, or catches & handles errors here?